packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +926,47 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting
+   * @param joinMeetingParams
+   */
+  export function joinMeeting(joinMeetingParams: JoinMeetingParams): Promise<string | object> {
```

Comment:
There should be new e2e tests for your function. Please add those to `meeting.json`. That file defines the e2e tests, and the test app that the e2e tests use has its existing meeting components defined in `MeetingAPIs.ts`.

In order for your e2e tests to work correctly you'll need to check in the other half of your changes to the host-sdk before these changes are checked in.

Reply:
In a side chat, you told me that these ids are definitly not UUIDs, and can take formats like `19:-D5VrQ35l94Drye2JbdTaiHiizqBAb4o_YnHlYtq5XM1@thread.tacv2`.